include/xframe/xaxis.hpp

```diff
@@ -594,6 +596,13 @@ namespace xf
     {
         return lhs.less_than(rhs);
     }
+
+    template <class T = std::size_t>
+    inline auto axis(T start, T stop, T step = 1) noexcept
```

Comment:
That should be
```cpp
template <class R = std::size_t, class T>
inlin auto axis(T start, T stop, T step = 1) noexcept
```

R being the label type, T can be deduced.

Reply:
T is already my label type, I should rename it to L then, and keep the same naming convention as in xaxis definition